core/.js/src/test/scala/io/udash/routing/RoutingEngineTest.scala

```diff
@@ -327,5 +328,44 @@ class RoutingEngineTest extends UdashFrontendTest with TestRouting {
       routingEngine.handleUrl(Url("/next"))
       renderer.views shouldBe Seq(rootView)
     }
+
+    "not rerender static views" in {
+      val staticView = new TestView
+      var staticCreateCount = 0
+      def staticViewFactory() = new StaticViewFactory[RootState](() => staticView) {
+        override def create(): (View, EmptyPresenter.type) = {
+          staticCreateCount += 1
+          super.create()
+        }
+      }
+      val state2VP: Map[TestState, () => ViewFactory[_ <: TestState]] = Map[TestState, () => ViewFactory[_ <: TestState]](
```

Comment:
Nit (applies to the line 19 too): one of explicit type declarations is unnecessary.